openlab/test-report.adoc

```diff
@@ -114,11 +114,13 @@ cyclictest -l10000 -m -Sp90 -i200 -h400 -q
 ----
 ===== Hypervisor
 image::cyclictest_results_hyp.png[]
+include::include/cyclictest_hyp.adoc[]
 
 <<<
 ===== Virtual Machines
 ====== Guest 0
 image::cyclictest_results_vm.png[]
+include::include/cyclictest_vm.adoc[]
```

Comment:
Put this one in optional : 
`include::include/cyclictest_vm.adoc[opts=optional]`

If the CI fails during the test of the hypervisor this file will not be created, but we to create the report.